cocos2d/core/components/CCComponent.js

```diff
@@ -247,9 +251,11 @@ var Component = cc.Class({
     /**
      * !#en
      * When attaching to an active node or its node first activated.
-     * onLoad is always called before any start functions, this allows you to order initialization of scripts.
+     * onLoad is always called before any start functions, this allows you to order initialization of scripts.<br/>
+     * PS: The method is a lifecycle method, and the base class may not implemente it. You can only call its base class implementation within the method, and you cannot call the method directly elsewhere.
```

Comment:
去掉 PS
This is a lifecycle method. It may not be implemented in the super class. You can only call its super class method inside it. It should not be called manually elsewhere.